### PR TITLE
Add fake module sample to test JS, CSS and Twig files

### PR DESCRIPTION
**Pull request by matks**
As said in https://github.com/PrestaShopCorp/header-stamp/pull/23, I wanted to cover also JS, CSS and Twig files to make sure we can modify the code of header-stamp without worrying of introducing a regression.

Here it is.

I will then miss the "vue" and "json" usecases 🤔 do you know modules we could use as samples ?

### CODE REVIEW

File: tests/integration/runner/run.php [hunk 1]
```diff
@@ -13,6 +13,7 @@
 $modulesToTest = [
     'gsitemap',
     'dashproducts',
+    'fakemodule',
```
Comment (matks) [inline tests/integration/runner/run.php:16]: This is needed anytime we add a sample module. Maybe I should remove this hardcoded list and parse the module-samples folder instead ?

(maybe a new PR 😄 )

File: tests/integration/module-samples/fakemodule/translations/cs.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright 2003-2020 PrestaShop SA
```
Comment (matks) [inline tests/integration/module-samples/fakemodule/translations/cs.php:23]: I put intentionally some crazy stuff to make sure header-stamp fixes it correctly 😄

**Comment by Quetzacoalt91** [comment]
> I will then miss the "vue" and "json" usecases thinking do you know modules we could use as samples ?

We could use ps_checkout or ps_analytics to test the feature about vue files.

Note all the json files aren't actually modified. Only `composer.json` and `package.json` are concerned, with the update of `author` and `license` keys

**Comment by matks** [comment]
Merge 🙏  ?

**Comment by matks** [comment]
Thank you for review 🎉

**Pull request merged**